### ISSUE TITLE
lambda: ZipFile param is not taken into account

### ISSUE THREAD

**Issue by pouledodue** [spulec/moto#713]
```
    conn.create_function(
        FunctionName='testFunction',
        Runtime='python2.7',
        Role='arn:aws:iam::00000000000:role/test-iam-role',
        Handler='lambda_function.handler',
        Code={
            'ZipFile': lambda_test_function_zip_content, <======== does nothing
        }
```

a call to `conn.invoke()` always returns response with a payload of `{result: "Good"}`

**Comment by rocky4570** [comment]
I have some code I have developed that allows some rudimentary python to be executed and the results returned on RequestResponse. Eg moto to invoke a boto call to get a volume from moto - if that makes sense?

**Comment by spulec** [comment]
Thanks for opening. This was fixed with #723

**Comment by pouledodue** [comment]
great, thanks!!
